test/functionality/dataflow/processing-of-elements/functions/source-tests.ts

```diff
@@ -1,21 +1,14 @@
 import {assertDataflow, withShell} from '../../../_helper/shell'
-import {setSourceFileProvider} from '../../../../../src/dataflow/internal/process/functions/source'
+import { setTextSourceProvider} from '../../../../../src/dataflow/internal/process/functions/source'
 import {BuiltIn, DataflowGraph, EdgeType, initializeCleanEnvironments} from '../../../../../src'
 import {LocalScope} from '../../../../../src/dataflow/environments/scopes'
 import {UnnamedArgumentPrefix} from '../../../../../src/dataflow/internal/process/functions/argument'
 import {define} from '../../../../../src/dataflow/environments'
 
 describe('source', withShell(shell => {
-	const sources = new Map<string, string>([
+	setTextSourceProvider(new Map<string, string>([
```

Comment:
using varargs we could get rid of the map construction (at least in the tests :D)
just passing an object like:
`{ simple: 'N <- 9' }` should be more readable but this is not a must-have.